cookiecutter.json

```
@@ -15,13 +15,23 @@
     "GNU General Public License v3",
     "Other"
   ],
-  "command_line_interface": ["Click", "Argparse", "No command-line interface"],
+
+  "command_line_interface": [
+    "Click", 
+    "Argparse", 
+    "No command-line interface"
+  ],
+  "doc_generator":[
+    "mkdocs",
+    "Sphinx",
+    "Jupyter-book"
+  ],
   "test_library": [
     "pytest"
   ],
-  "git_username": "zoro",
-  "git_https_origin": "Git remote origin (if known)",
-  "git_https_upstream": "Git remote upstream (if known)",
+  "git_username": "zoro_roronoa",
+  "git_https_origin": "https:// Git remote origin (if known)",
+  "git_https_upstream": "https:// Git remote origin (if known)",
```

this is upstream not origin ... but I think that it would be better to keep it empty instead

```suggestion
  "git_https_upstream": "",
```